src/zc/buildout/buildout.py

```diff
@@ -110,6 +110,8 @@ def overrideValue(self, sectionkey):
         self.value = sectionkey.value
         if sectionkey.history[-1].operation not in ['ADD', 'REMOVE']:
             self.addToHistory("OVERRIDE", sectionkey.value, sectionkey.source)
+        else:
+            self.history = copy.deepcopy(sectionkey.history)
```

Comment:
deepcopy() always scares me.  I did not investigate whether its use is safe here.

Reply:
AFAIU, the issues with `deepcopy` do not apply here as `history` is a list of `HistoryItem` instances which only holds strings.